src/pages/api/sepana/delete.page.ts

```diff
@@ -0,0 +1,45 @@
+import { deleteRecord } from 'lib/sepana'
```

Comment:
These APIs are all publicly facing - do we want to put some auth on them?

Reply:
they are all authenticated using private server keys. there's no harm in calling any except the delete endpoints, which require an admin key that won't be included in the public build (and isn't in .example.env)

i'm open to some added security layer, but worth noting that even deleting the entire DB would hardly be a problem—it would just get repopulated within a few minutes the next time /update gets called, and queries would be broken until then. sepana shouldn't be a source of truth for anything

Reply:
Sounds GTM!